lib/elixir/test/elixir/record_test.exs

```diff
@@ -137,7 +137,9 @@ defmodule RecordTest do
     charlist: 'abc'
 
   test "records with literal defaults" do
-    assert defaults(defaults()) == [
+    defaults = defaults()
```

Comment:
Maybe a comment here saying that we need to have this on its own line to detect any escaping issues which would be by passed with we just passed it to the macro below?